src/field/field.cxx

```diff
@@ -62,5 +61,6 @@ void Field::error(const char *s, ...) const {
   }
   
   throw BoutException("Error in field: %s", err_buffer);
+  delete[] err_buffer;
```

Comment:
This will never be hit, and presumably leads to a memory leak if the exception isn't caught?
I'm not really sure a) if this is a problem and b) if so, how to fix it.